[bug] org.openqa.selenium.TimeoutException 문제 발생

## Describe the bug
https://tech.kakao.com/posts/455
이 url에는 image_url이 존재함.
image_url을 계속해서 가져오지 못해 서버 로그를 확인해본 결과 TimeoutException 발생

```java
baguni.common.lib.opengraph.OpenGraphException: Error occurred when reading OG tags via Selenium, url : https://tech.kakao.com/posts/455

Caused by: org.openqa.selenium.TimeoutException: timeout: Timed out receiving message from renderer: 9.821
  (Session info: headless chrome=114.0.5735.106)
Build info: version: '4.19.1', revision: 'abe0ee07dc'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '6.8.0-47-generic', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e649e97f0f2be10c31257f041bdf8215, get {url=https://tech.kakao.com/posts/455}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 114.0.5735.106, chrome: {chromedriverVersion: 114.0.5735.90 (386bc09e8f4f..., userDataDir: /tmp/.com.google.Chrome.DG857b}, goog:chromeOptions: {debuggerAddress: localhost:35227}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:cdp: ws://localhost:35227/devtoo..., se:cdpVersion: 114.0.5735.106, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: e649e97f0f2be10c31257f041bdf8215
```
<!-- 버그가 무엇인지 명확하게 설명해주세요. -->

--- a/backend/baguni-common/src/main/java/baguni/common/config/OpenGraphConfig.java
+++ b/backend/baguni-common/src/main/java/baguni/common/config/OpenGraphConfig.java
@@ -10,6 +10,6 @@ public class OpenGraphConfig {
 
 	@Bean
 	public OpenGraphOption openGraphOption() {
-		return new OpenGraphOption(10);
+		return new OpenGraphOption(2 * 60);
 	}
 }

--- a/backend/baguni-common/src/main/java/baguni/common/lib/opengraph/OpenGraphReaderSelenium.java
+++ b/backend/baguni-common/src/main/java/baguni/common/lib/opengraph/OpenGraphReaderSelenium.java
@@ -91,6 +91,9 @@ public class OpenGraphReaderSelenium implements OpenGraphReader {
 					result.put(nameAttr, value);
 				}
 			}
+		} catch (org.openqa.selenium.TimeoutException e) {
+			log.error("Selenium TimeoutException 발생 : {}, url : {}", e.getMessage(), uri);
+			// result = new OpenGraphReaderJsoup(openGraphOption).read(uri); // 추후 Jsoup 사용 시 이 코드 활용
 		} catch (Exception e) {
 			throw new OpenGraphException("Error occurred when reading OG tags via Selenium, url : " + uri, e);
 		} finally {

